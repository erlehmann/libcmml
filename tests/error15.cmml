<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<stream id="a1" basetime="0">
<import contenttype="audio/x-wav" src="00-3-18.wav" start="npt:0"/>
</stream>

<head lang="en">
<title>Buried dams of water</title>
</head>

<clip id="Introduction" start="npt:0:00:06.20">
<a href="99-5-8.anx#Introduction">
A related project that studies the bacteria growing in this stored water</a>
<desc>
Water storage research: Storing water underground! It's
like a dam that contains excess water which can then be used at
times of high demand. Simon Toze of CSIRO Land and Water says
it's being done using existing aquifers, but it has to be
carefully planned.
</desc>
</clip>

<clip id="nonsequential" start="npt:0:00:00.20">
<desc>
This is a clip that starts before the one above and should cause an
error in non-sloppy parsing mode.
</desc>
</clip>

</cmml>
