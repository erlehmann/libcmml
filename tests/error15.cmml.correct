<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<stream id="a1" basetime="0.000">
<import contenttype="audio/x-wav" src="00-3-18.wav" start="npt:0.000"/>
</stream>

<head lang="en">
<title>Buried dams of water</title>
</head>

<clip id="Introduction" track="default" start="npt:0:00:06.200">
<a href="99-5-8.anx#Introduction">
A related project that studies the bacteria growing in this stored water</a>
<desc>
Water storage research: Storing water underground! It's
like a dam that contains excess water which can then be used at
times of high demand. Simon Toze of CSIRO Land and Water says
it's being done using existing aquifers, but it has to be
carefully planned.
</desc>
</clip>

