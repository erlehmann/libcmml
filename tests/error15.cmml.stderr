error15.cmml:26:0: Skipped clip out of temporal order
